MonoGame.Framework/Graphics/Effect/IEffectFog.cs

```diff
@@ -48,32 +10,31 @@ namespace Microsoft.Xna.Framework.Graphics
 	public interface IEffectFog
 	{
         /// <summary>
-        /// The floating point fog color.
+        /// Gets or set the fog color in floating point format.

```

Comment:
> Wast majority of classes uses this notation

A search shows that only 61 classes/structs/interfaces use "Gets or sets" in them.